docs/decisions/0003-theming-using-webpack-module-federation.rst

```diff
@@ -0,0 +1,125 @@
+Runtime Theming Using WebPack Module Federation
+=============================================
+
+Status
+------
+
+Proposal
+
+Context
+-------
+
+Webpack 5 introduced a new feature called "Module Federation" which allows
```

Comment:
Please link `Module Federation` to https://webpack.js.org/concepts/module-federation/

Reply:
It's listed at the end under references.